libkbfs/tlf_edit_history.go

```diff
@@ -528,6 +556,36 @@ func (teh *TlfEditHistory) updateHistory(ctx context.Context,
 	}
 
 	currEdits.addNewEdits(newEdits)
+	// Send the notifications.
+	for writer, edits := range newEdits {
+		for _, edit := range edits {
+			var n *keybase1.FSNotification
+			switch edit.Type {
+			case FileCreated:
+				cop, ok := edit.op.(*createOp)
+				if !ok {
+					return fmt.Errorf("No create op for create notification, "+
```

Comment:
Yes, effectively.  I generally try to avoid panics in production code unless we're trying to debug a rare problem and need a stacktrace. Just in case.  I could probably be convinced otherwise though if you have a strong argument.